content/source/docs/enterprise/workspaces/repo-structure.html.md

```diff
@@ -72,25 +49,15 @@ workspace — one set of variables for `prod` and one set for `stage`.
 │   │   ├── outputs.tf
 ```
 
-Each workspace listens to a specific
-branch for changes, as configured by the [VCS branch setting](./settings.html#vcs-branch).
-The `networking-prod` workspace should be configured to listen to the `prod`
-branch and `networking-stage` to `stage.` To promote a change to stage, just open
-a PR against that branch. To promote to prod, open a PR for stage against prod.
+Each workspace listens to a specific branch for changes, as configured by the [VCS branch setting](./settings.html#vcs-branch). This means that plans will not occur in a given workspace until a PR is opened against or merged into the designated branch. The `networking-prod` workspace would be configured to listen to the `prod` branch, `networking-stage` to `stage`, and `networking-dev` to `dev`. To promote a change to `stage`, just open a PR against the `stage` branch. To promote to prod, open a PR from `stage` against `prod`.
```

Comment:
plans will also occur for `commit push` or `tag push` to the source branch (we can use `push event` or `push` to grab these wholesale)

Reply:
How does "until a PR is opened or a push event occurs" sound?